clients/pay-client/lib/pay_gov_au/model/create_request_payment.ex

```diff
@@ -22,13 +23,17 @@ defmodule PayGovAu.Model.CreateRequestPayment do
           :description => String.t() | nil,
           :email => String.t() | nil,
           :gateway_account_id => integer() | nil,
+          :metadata => Map | nil,
```

Comment:
not sure if capitalisation matters here, usually set as `map`

https://hexdocs.pm/elixir/typespecs.html#basic-types

Reply:
@dagoof interesting. This is codegen from the open API client generator: https://github.com/OpenAPITools/openapi-generator/blob/cc1bfe5fef4087be4310168972cc7bcfe1da7461/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElixirClientCodegen.java#L174

Seems to work though. I added some metadata to the products app when creating the payment and it worked at least.